internal/pithosctl/pkg/kubernetes/kubernetes.go

```diff
@@ -70,6 +70,16 @@ func (c *Client) Pods(selector, namespace string) ([]v1.Pod, error) {
 	return podList.Items, nil
 }
 
+// PithosConfigMap return configmap containing pithos configuration
+func (c *Client) PithosSecret(secretName, namespace string) (*v1.Secret, error) {
```

Comment:
There doesn't seem to be anything Pithos-specific about this function, so I'd just call it `Secret` or `GetSecret`.